packages/libs/components/.storybook/main.js

```diff
@@ -25,7 +25,11 @@ module.exports = {
       test: /\.(js|jsx)$/,
       loader: require.resolve('babel-loader'),
       options: {
-        plugins: ['@babel/plugin-proposal-nullish-coalescing-operator'],
+        plugins: [
+          '@babel/plugin-proposal-nullish-coalescing-operator',
+          '@babel/plugin-syntax-class-properties',
```

Comment:
Why was this added? I don't see class properties used anywhere.

Reply:
Thank you!! I forgot to ask about this in our PR review! I was curious since it's scoped to storybook.

Reply:
ahh yes good catch! This was a byproduct of my adventures trying to install some of the visx packages a while back. I just removed it and all seems to work  👍